ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -57,7 +57,7 @@
           wipe-workspace: true
 
     builders:
-      - shell: "export NPROC=$(nproc); timeout 7200 ./run-make-check.sh"
+      - shell: "export NPROC=$(nproc) CHECK_MAKEOPTS='-j$(nproc) -N'; timeout 7200 ./run-make-check.sh"
```

Comment:
how about adding `-Q` to silence the output?

Reply:
Done